doc/18_Tools_and_Features/25_System_Settings.md

```diff
@@ -1,8 +1,30 @@
 # System Settings
 
 In the system settings (*Settings* > *System Settings*) system-wide settings for Pimcore can be made. Changes should 
-be made with care and only by developers. 
-These settings are saved in `var/config/system_settings/system_settings.yaml` or in the settings store based on your configuration.
+be made with care and only by developers.
+
+These settings are saved in `var/config/system_settings/system_settings.yaml` or in the settings store based on your configuration. The production environment has them stored in the settings store by default.
+
+To save system settings into the settings store, you will need to add following to your configuration:
+```yaml
+pimcore:
+    config_location:
+        system_settings:
+            write_target:
+                type: 'settings-store'
+            read_target:
+                type: 'settings-store'
+```
+
+To switch from Symfony configuration to settings store for the first time, you need to copy the content of your `system_settings.yaml` file and convert it into JSON format.
```

Comment:
well, that is not entirely correct ... you also could set read_target to symfony-config and write-target to settings-store ... and then just save the settings once. 

that might be easier that the procedure described here.